app/models/message.rb

```diff
@@ -233,6 +234,19 @@ def ensure_processed_message_content
     self.processed_message_content = message_content&.truncate(150_000)
   end
 
+  # fetch the in_reply_to message and set the external id
+  def ensure_in_reply_to_external_id
+    in_reply_to = content_attributes['in_reply_to']
```

Comment:
```suggestion
    in_reply_to = content_attributes&.['in_reply_to']
```

Reply:
@pranavrajs had to revert this, the linter was complaining

![CleanShot 2023-10-12 at 22 04 16](https://github.com/chatwoot/chatwoot/assets/18097732/2e70ec9f-22ba-41eb-b82c-f58c00290394)